tests/test_insert.py

```diff
@@ -23,3 +24,40 @@ def test_insert_into_with_columns_and_select_union():
         {"tab2", "tab3"},
         {"tab1"},
     )
+
+
+def test_insert_with_custom_columns():
+    # test with query as subquery
+    sql = "insert into trg_tbl(random1,random2) (select col1,col2 from src_tbl)"
+    assert_column_lineage_equal(
+        sql,
+        [
+            (
+                ColumnQualifierTuple("col1", "src_tbl"),
+                ColumnQualifierTuple("random1", "trg_tbl"),
+            ),
+            (
+                ColumnQualifierTuple("col2", "src_tbl"),
+                ColumnQualifierTuple("random2", "trg_tbl"),
+            ),
+        ],
+        test_sqlparse=False,
```

Comment:
skipping the sqlparse for all the tests as sqlparse either does not generate or generates incorrect column level lineage as described in pr description